loadbalancer/src/main/java/com/io/loadbalancer/consistent_hashing/HashingConfiguration.java

```diff
@@ -28,16 +41,16 @@ public PopularityMonitor popularityMonitor(){
                 IMMax);
     }
 
+    @Bean
+    public ImageManagerMonitor imageManagerMonitor() {
+        return new ImageManagerMonitor(getImageManagerList(), monitorClient);
+    }
+
     @Bean
     public Hashing hashing(){
-        int IMMax = properties.getImageManagers().size();
         return new Hashing(
-                properties
-                        .getImageManagers()
-                        .stream()
-                        .map(manager -> new ImageManager(manager.getUrl()))
-                        .collect(Collectors.toList()),
+                getImageManagerList(),
                 popularityMonitor(),
-                new ImageManagerMonitor(IMMax)); // Not sure if it should be initialized here
+                imageManagerMonitor());
```

Comment:
Could you please ensure that when no instance is found running the Hashing class will return 500 or something like this?

Reply:
Isn't it already handled by exception in Hashing class? Later the exception is caught in the Balancer class.

Reply:
That's why I asked to ensure that it is implemented :)